ui/lib/pki/addon/components/page/pki-configuration-edit.hbs

```diff
@@ -123,39 +123,35 @@
       {{/if}}
     </fieldset>
 
-    <div class="field is-grouped box is-fullwidth is-bottomless">
-      <div class="control">
-        {{#if (or @urls.canSet @crl.canSet)}}
-          <button
-            type="submit"
-            class="button is-primary {{if this.save.isRunning 'is-loading'}}"
-            disabled={{this.save.isRunning}}
-            data-test-configuration-edit-save
-          >
-            Save
-          </button>
-        {{/if}}
-        <button
-          {{on "click" this.cancel}}
-          type="button"
-          class="button has-left-margin-s"
+    <hr class="has-background-gray-100" />
+    <Hds::ButtonSet>
```

Comment:
🎉 even though there's a conditional wrapping one of the buttons - I think this is the right move!